### PR TITLE
modifiy spaces and icons

### LINKED ISSUE #55: UI changes - Menu indentation
Left menu should respect some indentation rule
Example of what should be indented:

Screenshot from 2022-06-17 12-23-34

### PR DESCRIPTION
**Pull request by samaradel**
Add spaces between parents and children.

image

image

image

image

Related issues:

[#55](https://github.com/threefoldtech/tfgrid_dashboard/issues/55)
[#57](https://github.com/threefoldtech/tfgrid_dashboard/issues/57)

### CODE REVIEW

**Review by MohamedElmdary — APPROVED** [review]

File: src/portal/views/Dao.vue [hunk 1]
```diff
@@ -34,90 +29,88 @@
...
+              }}/{{ proposal.threshold }}
             </span>
             <v-divider vertical />
-
           </div>
           <v-btn
-            color="red"
-            :width="`${proposal.nayes.length *100 + 100}`"
+            color="grey lighten-2 black--text"
+            width="100px"
```
Comment (amiraabouhadid) [inline src/portal/views/Dao.vue]: hey, @samaradel please do not remove the width related to the number of ayes and nays on a proposal as it should display as the proposal's nays and ayes are trying to reach the threshold.

File: src/portal/views/Dao.vue [hunk 2]
```diff
@@ -34,90 +29,88 @@
...
-
-            <v-col
-              :sm="Math.round(proposal.ayesProgress*12/100)===12? 11: proposal.ayesProgress*12/100"
+            <v-progress-linear
+              :value="proposal.ayesProgress"
               height="25"
-              style="background-color: green;"
+              :style="{
+                width: proposal.ayesProgress + '%',
+                marginRight: 'auto',
```
Comment (amiraabouhadid) [inline src/portal/views/Dao.vue:99]: Hey @samaradel , please make the progress bar green for ayes and red for nays so there is a clearer visualization of which is winning the vote

File: src/portal/views/Dao.vue [hunk 3]
```diff
@@ -34,90 +29,88 @@
...
-
         </v-container>
-        <v-container>
+        <v-container
+          v-if="proposal.ayesProgress > 0 || proposal.nayesProgress > 0"
+        >
           <v-row justify="center">
-
-            <v-col
-              :sm="Math.round(proposal.ayesProgress*12/100)===12? 11: proposal.ayesProgress*12/100"
```
Comment (amiraabouhadid) [inline src/portal/views/Dao.vue:101]: Please also do not remove this as it is used to calculate and visualize the progress based on vote weights

File: src/portal/views/Dao.vue [hunk 4]
```diff
@@ -34,90 +29,88 @@
...
+              <template>
+                <strong>{{ proposal.ayesProgress }}%</strong>
+              </template>
+            </v-progress-linear>
+            <v-progress-linear
+              :value="proposal.nayesProgress"
+              color="grey lighten-2"
               height="25"
-              style="background-color: red;"
-              :sm="Math.round(proposal.nayesProgress*12/100)===12? 11: proposal.nayesProgress*12/100"
```
Comment (amiraabouhadid) [inline src/portal/views/Dao.vue:110]: please leave the calculation here as well to allow visualization of nayes progress as such
Screenshot from 2022-06-28 13-49-51

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]
Hey @samaradel , please do not remove the logic used to calculate the progress bar width/color and the Yes/No buttons width. It needs to look like this: 

Screenshot from 2022-06-28 13-49-51
Screenshot from 2022-06-28 13-52-07

**Comment by amiraabouhadid** [comment]
@samaradel Please also link all the issues you have addressed to this pull request so that they are automatically closed.
Thanks,
Amira

**Comment by samaradel** [comment]
@amiraabouhadid all resolved now

File: src/portal/views/Dao.vue [hunk 5]
```diff
@@ -38,90 +33,85 @@
...
+              >Threshold: <br />{{
+                proposal.nayes.length + proposal.ayes.length
+              }}/{{ proposal.threshold }}
             </span>
             <v-divider vertical />
-
           </div>
           <v-btn
-            color="red"
-            :width="`${proposal.nayes.length *100 + 100}`"
```
Comment (amiraabouhadid) [inline src/portal/views/Dao.vue:86]: @ehab-hassan @samaradel please include this

File: src/portal/views/Dao.vue [hunk 6]
```diff
@@ -38,90 +33,85 @@
...
 
-        <v-container
-          fluid
-          class="d-flex justify-space-between my-5"
-        >
+        <v-container fluid class="d-flex justify-space-between my-5">
           <v-btn
-            color="green"
-            class=" "
-            :width="`${proposal.ayes.length *100 + 100}`"
```
Comment (amiraabouhadid) [inline src/portal/views/Dao.vue:66]: @samaradel @ehab-hassan please include this

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]

File: src/portal/views/Dao.vue (hunk 6 above)
Comment (amiraabouhadid) [reply src/portal/views/Dao.vue:66]: fixed it, thanks

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**